src/components/Proposal/View/BiddingAndTenderingProcess.tsx

```diff
@@ -1,37 +1,124 @@
 import React, { useMemo } from 'react'
 
 import useFormatMessage from 'decentraland-gatsby/dist/hooks/useFormatMessage'
+import Time from 'decentraland-gatsby/dist/utils/date/Time'
 
-import { ProposalType } from '../../../entities/Proposal/types'
+import { ProposalAttributes, ProposalStatus, ProposalType } from '../../../entities/Proposal/types'
+import useProposal from '../../../hooks/useProposal'
 
-import ProposalProcess from './ProposalProcess'
+import ProposalProcess, { ProcessStatus } from './ProposalProcess'
 
 interface Props {
+  proposalId?: ProposalAttributes['id']
   proposalType: ProposalType
+  proposalStatus: ProposalStatus
+  proposalFinishAt: ProposalAttributes['finish_at']
+  linkedProposalId?: string
+  tenderProposalsTotal?: number
 }
 
-export default function BiddingAndTenderingProcess({ proposalType }: Props) {
+const getPitchConfig = (type: ProposalType, status: ProposalStatus) => {
+  if (type === ProposalType.Pitch && status === ProposalStatus.Active) {
```

Comment:
I think this portion of code might be clearer if you group first by type and then by status 

i.e.:

```typescript
if (type === ProposalType.Pitch) {
   if(status === ProposalStatus.Active) {
      ...
   }
   if(status === ProposalStatus.Rejected) {
      ...
   }
}
if (type === ProposalType.Tender) {
   ...
}
```